playwright-tests/tests/community.spec.js

```diff
@@ -43,6 +43,7 @@ test.describe("Wallet is connected", () => {
   test("should allow connected user to post from community page", async ({
     page,
   }) => {
+    test.setTimeout(60000);
```

Comment:
just to confirm, have you added it because of the RPC issue?

Reply:
yes I observed that the tests where I've increased the timeout often fails because of timeout. The default is 30000, so increasing it to 60000 should give it more time to finish.